README.md

```diff
@@ -207,11 +208,11 @@ Just specify a GitHub repository in the URL and off you go!
 - [ ] add GitHub search
 
 
-[gh-board-image]: https://img.shields.io/github/issues/philschatz/gh-board.svg?label=Issues%20%28gh-board%29
-[gh-board-url]: http://philschatz.com/gh-board/
-[travis-image]: https://img.shields.io/travis/philschatz/gh-board.svg
-[travis-url]: https://travis-ci.org/philschatz/gh-board
-[dependency-image]: https://img.shields.io/david/philschatz/gh-board.svg
-[dependency-url]: https://david-dm.org/philschatz/gh-board
-[dev-dependency-image]: https://img.shields.io/david/dev/philschatz/gh-board.svg
-[dev-dependency-url]: https://david-dm.org/philschatz/gh-board?type=dev
+[issues-image]: https://img.shields.io/github/issues/coala/gh-board.svg?label=Issues%20%28gh-board%29
+[issues-url]: http://philschatz.com/gh-board/
```

Comment:
This needs to be our repo's url, https://github.com/coala/gh-board/issues

Reply:
Alright, done. Thanks for helping out :)